app/mailers/covid19_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class Covid19Mailer < ApplicationMailer
+  def covid_19_delay(application_form)
```

Comment:
Nitpick, thinking `covid_19` is already part of the mailer name. Feel free to ignore!

```suggestion
  def delay(application_form)
```

Reply:
Agreed, reworked the method name and the location of the preview code.